### PR TITLE
Integrate veBalFeeInjector into fee redirection logic

### PR DESCRIPTION
**Pull request by Tritium-VLK**
1: Cowswap final USD directly injector
2: Send BAL to injector when placing the cowswap order.

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Will merge this before the run and do final testing/hammer out anything I missed when we do the fee sweep on Friday.  Regardless, would be good to get some extra eyes on these changes and make sure they are in place for Friday.

What this basically does is:

1: Change Swap of remaining USDC for bb-a-usd to send it directly to the injector
2: Send the balance of BAL in the safe to the injector.

This will all happen in the same tx where all the bribs/transfers are paid.

File: tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py [hunk 1]
```diff
@@ -4,10 +4,13 @@
 from brownie import web3, interface
 from web3 import Web3
 from great_ape_safe import GreatApeSafe
-from helpers.addresses import r
+from bal_addresses import AddrBook
 import csv
 from datetime import date
 
+a = AddrBook("mainnet")
```
Comment (SHAKOTN) [inline tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py:11]: Please, avoid naming variables with one letter. Especially global ones

File: tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py [hunk 2]
```diff
@@ -188,9 +190,11 @@ def bribe_balancer(gauge, mantissa):
             mantissa,  # uint256 amount
         )
 
-    print(f"Swapping leftover USDC for {veBalFeeToken}")
+    print(f"Swapping leftover USDC for {veBalFeeToken} and sending fees to the injector")
     usd = safe.contract(veBalFeeToken)
-    safe.cow.market_sell(usdc, usd, usdc.balanceOf(safe.address), 8*60*60, 1, 0.995)
+    bal = safe.contract(r.tokens.BAL)
+    safe.cow.market_sell(usdc, usd, usdc.balanceOf(safe.address), 8*60*60, 1, 0.995, r.maxiKeepers.veBalFeeInjector)
```
Comment (SHAKOTN) [inline tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py:196]: Move magic numbers into constants on the top of the file

File: tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py [hunk 3]
```diff
@@ -188,9 +190,11 @@ def bribe_balancer(gauge, mantissa):
             mantissa,  # uint256 amount
         )
 
-    print(f"Swapping leftover USDC for {veBalFeeToken}")
+    print(f"Swapping leftover USDC for {veBalFeeToken} and sending fees to the injector")
     usd = safe.contract(veBalFeeToken)
```
Comment (SHAKOTN) [inline tools/python/brownie/scripts/maxi_operations/bribe_ecosystems.py:194]: Please rename this into something more meaningful? Like, `fee_token`

**Pull request merged**